### DISCUSSION TITLE
Disable the Send API Request button

### DISCUSSION THREAD

**Discussion by cryptodam** [PaloAltoNetworks/docusaurus-openapi-docs#535] [Q&A]
Hello, 
Is there a way to disable the Send API Request for some API endpoints and not other. 
Other question
Is it possible to disable it for some Urls and not others. 

In the end I would like to do the following:
- EndPoint1
  - url1 : Enabled
  - url2 : Enabled

- Endpoint2
  - url1 : Enabled
  - url2 : Disabled

**Comment by sserrata** [comment]
Hi @cryptodam, you can disable by endpoint but disabling by URL is currently not supported. 

> Use the `hide_send_button: true` frontmatter to disable by endpoint/doc.

**Comment by sserrata** [comment]
Btw, you can try to swizzle the `@theme/ApiDemoPanel/Execute` component to customize this behavior.
